anymatrix.m

```diff
@@ -310,6 +310,40 @@
         IDs = IDs.';
     end
 
+
+    % Try to run function setup() if current group has one.
+    function run_setup_function_if_repository_changed(changed)
+        run_function_if_repository_changed(changed, 'setup');
+    end
+
+    % Try to run function update() if current group has one.
+    function run_update_function_if_repository_changed(changed)
+        run_function_if_repository_changed(changed, 'update');
+    end
+
+    % If current group has a function called `function_name`, run it after
+    % asking user for confirmation.
+    function run_function_if_repository_changed(changed, function_name)
+        if changed && exist([function_name '.m'], 'file')
+            disp('Setting up and updating this group requires running a function.');
+            repeat = true;
+            while (repeat)
+                choice = input('Do you wish to continue? [Y/n]: ','s');
```

Comment:
Could we just go ahead and run the setup/update instead of asking the user to say y/n? If they are running the anymatrix remote group command, they are probably already expect one of those two things to happen.

Reply:
I've removed the checks, and I've only retained the message saying that a function is being run.